src/ui/welcome/index.html

```diff
@@ -9,12 +9,14 @@
 			--button-hover-background: rgba(0,0,0,0.1);
 			--inversion: 1; /* logo is white background, light themes need an inversion */
 			--header-colour: #555555;
+			--command-background : rgba(255,255,255,0.08);
 		}
 		.vscode-dark{
 			--button-background: rgba(255,255,255,0.08);
 			--button-hover-background: rgba(255,255,255,0.1);
 			--inversion: 0;
 			--header-colour: #e0b009;
+			--command-background : black;
```

Comment:
Why is there an opacity in light theme, but none in dark theme for `--command-background`?